app/assets/stylesheets/pages/about.scss

```diff
@@ -111,139 +125,139 @@
 }
 
 .hover-icon {
-  position:relative;
-}
+  position: relative;
 
-.hover-icon img {
-  position: absolute;
-  left: 0;
-  @include transition(opacity 0.3s ease-in-out);
-}
+  img {
+    @include transition(opacity 0.3s ease-in-out);
+    position: absolute;
+    left: 0;
 
-.hover-icon img.icon-top:hover {
-  opacity: 0;
+    &.icon-top:hover {
+      opacity: 0;
+    }
+  }
 }
 
-/** NAVIGATION ARROWS */
-
+// NAVIGATION ARROWS
 .nav-circlepop {
   position: relative;
-}
 
-.nav-circlepop a {
-  margin: -42px 14px;
-  width: 50px;
-  height: 50px;
-}
+  a {
+    margin: -42px 14px;
+    width: 50px;
+    height: 50px;
 
-.nav-circlepop a::before {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  border-radius: 50%;
-  background: #fff;
-  content: '';
-  opacity: 0;
-  @include transition(transform 0.3s);
-  @include transition(opacity 0.3s);
-  @include transform(scale(0.9));
-}
-
-.nav-circlepop .icon-wrap {
-  position: relative;
-  display: block;
-  margin: 32% 0 0 33%;
-  width: 80%;
-  height: 80%;
-}
+    &.next .icon-wrap {
+      @include transform(rotate(90deg));
+    }
 
-.nav-circlepop a.next .icon-wrap {
-  @include transform(rotate(90deg));
-}
+    &::before {
+      @include transition(transform 0.3s);
+      @include transition(opacity 0.3s);
+      @include transform(scale(0.9));
+
+      background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#fff` should be written as `#FFF`